packages/py-api/py_api/controllers/hackathon_participants_controller.py

```diff
@@ -0,0 +1,69 @@
+import json
+
+from bson.json_util import dumps
+from bson.objectid import ObjectId
+from fastapi.responses import JSONResponse
+from py_api.database.initialize import participants_col
+from py_api.models import RandomParticipant
+from py_api.utilities.parsers import has_prohibited_characters
+
+
+class PartcipantsController:
+
+    @classmethod
+    def get_all_participants(cls) -> JSONResponse:
+        participants = list(participants_col.find())
+
+        if not participants:
+            return JSONResponse(content={"message": "No participants were found!"}, status_code=404)
+
+        # the dumps funtion from bson.json_util returns a string
+        return JSONResponse(content={"participants": json.loads(dumps(participants))}, status_code=200)
+
+    @classmethod
+    def get_specified_participant(cls, object_id: str) -> JSONResponse:
+        specified_participant = participants_col.find_one(
+            filter={"_id": ObjectId(object_id)},
```

Comment:
Handle this 
![Problem with parcing ObjectID](https://github.com/AUBGTheHUB/monolith/assets/38375079/aa5941a2-1a88-4026-83e4-65d6f39209b6)